### ISSUE TITLE
Energy Meter gives false data

### ISSUE THREAD

**Issue by theunkn0wn1** [BluSunrize/ImmersiveEngineering#708]
The energy meter does not state the actual Energy transferred over the last 20 ticks, it states the Maximum Energy in *in a perfect system* over the last 20 ticks.

In the below system, the **maximum** transfer rate is 4096 RF/T (derived from configs) yet the **actual** energy transferred was 4090 RF/T at the meter. 
![image](https://cloud.githubusercontent.com/assets/3110986/11432204/b4dbf4a6-945d-11e5-9d69-e7a2626cee37.png)
Yet, when measurements where taken from the meter, it reported 4096 RF. That is simply false.
![image](https://cloud.githubusercontent.com/assets/3110986/11432214/f4f73e6a-945d-11e5-8be5-6d8b6c23605a.png) The average energy over the last 20 ticks, as the meter should see it, was 4090 RF. 

This issue is magnified over distance. For example, the below setup includes a distant power source that is a about 9 chunks away from the receiving capacitor. Due to energy loss, the capacitor is only receiving 7203 RF/T
![image](https://cloud.githubusercontent.com/assets/3110986/11432291/ee5c7a78-945f-11e5-893b-76e44c5b83d3.png)
Somehow the energy meter is only reading 4092 RF/T **// What?! //**
![image](https://cloud.githubusercontent.com/assets/3110986/11432303/239aee04-9460-11e5-8717-f95b400fa653.png)

Conclusion: The Energy meter is giving bad/incorrect data to players
===================

Ie build ae1d377

**Comment by BluSunrize** [comment]
It tells you the transfer without loss because I cannot calculate the loss to the exact subpoint of the connection.

**Comment by theunkn0wn1** [comment]
As for reading 4092 when the RF/T is 7203/t?

**Comment by BluSunrize** [comment]
Derp on my part, fixed already. I'll see if I can fix the loss thing somehow.

**Comment by theunkn0wn1** [comment]
OK. 

Could you not just calculate the distance between the Input and the meter, subtract that from the max throughput, and report that back to the user?

Also, whilst I'm here: Question on your Energy net: do connectors have a max output rate? if so, is it limited by the wire transfer rate or what?

**Comment by cloakable** [comment]
Connectors have a max output rate, wires have a higher transfer rate so you can have multiple feeding into one wire. Feed too much though and the wire will burn out.

**Comment by theunkn0wn1** [comment]
I know that the connectors have a max input rate, and that the cables max rate is several times the input rate, but during testing I found the connectors have no observable max output rate; I am attempting to determine if this is intentional before creating a issue just for it.

**Comment by cloakable** [comment]
...huh, I haven't noticed that. That's probably an issue, yes.

**Comment by BluSunrize** [comment]
Huh right, I guess I never thought about that in rewriting the energy system...

**Comment by theunkn0wn1** [comment]
Yeah, was leading to confusing results whilst bug hunting for this issue and a few other related ones. I have noticed that, when several inputs are wired into a single consumer, that consumer was taking in much more RF than expected. That is how i got such high numbers in my OP is that my mains had multiple HV inputs over a single line. a simplified setup is below:
![image](https://cloud.githubusercontent.com/assets/3110986/11435222/c141ddda-948c-11e5-89cf-27f5319135dc.png)

**Comment by BluSunrize** [comment]
Yeah, I know what's causing it.

**Comment by theunkn0wn1** [comment]
Poke @BluSunrize 
Just tested the attempt at loss calc... it seems to count the wrong way?
From testing with HV (4096 RF/T) I discovered that as the distance increases from source to the meter, the meter reports larger and larger "energy transferred in last 20 ticks".

In my testing, I discovered the meter reporting a transfer value of over 4096! This is impossible assuming this is the average energy transferred!This is false as the receiving capacitor only has one HV connector set to input, and it itself reads 4012 RF/T (Reasonable)

My setup is below:
![image](https://cloud.githubusercontent.com/assets/3110986/11493949/8bb23858-97b2-11e5-96e1-f35ac6be3df7.png)
The far meter (from image POV) reports energy flow of 4069 while the near meter (farther from the power source) reads 4085.
```4069<4085 = false```
The only thing reading a valid energy flow is the capacitor (from EIO) itself reading 3594 RF/T
